resources/views/pages/login.twig

```diff
@@ -107,11 +115,17 @@
                     {% if config('enable_password') %}
                         <p>{{ __('Please sign up, if you want to help us!') }}</p>
                         <a href="{{ url('register') }}" class="btn btn-primary">{{ __('Register') }} &raquo;</a>
+                    {% elseif config('external_register_url') %}
+                        <a href="{{ config('external_register_url') }}" class="btn btn-primary">{{ __('Register') }} &raquo;</a>
```

Comment:
At the moment this "doubles" the Register title above so we should either change the text or remove the title?
Also having multiple "buttons" in the code makes it imho not that greatly readable, so just changing the URL via short syntax would be imho preferrable here.